app/src/main/java/com/github/fribourgsdp/radio/PlayerGameHandler.kt

```diff
@@ -1,14 +1,17 @@
 package com.github.fribourgsdp.radio
 
+import android.content.Intent
 import android.util.Log
 import com.google.firebase.firestore.DocumentSnapshot
 
 class PlayerGameHandler(
-    private val gameID: Long,
+    val gameID: Long,
     private val view: GameView,
     db: Database = FirestoreDatabase()
 ): GameHandler(view, db), GameView.OnPickListener {
 
+    val database = db
```

Comment:
It wasn't really intended to be exposed...

Reply:
I unexposed it.